modules/components/src/DataTable/TableToolbar/TableToolbar.js

```diff
@@ -75,23 +78,48 @@ const TableToolbar = ({
         )}
       </div>
       <div className="group">
-        {allowTogglingColumns && (
-          <DropDown
-            aria-label={`Select columns`}
-            itemToString={(i) => i.Header}
-            items={canChangeShowColumns}
-            onChange={(item) => {
-              setFilterVal('');
-              onFilterChange({
-                value: '',
-                generateNextSQON: generateNextSQON(''),
-              });
-              onColumnsChange({ ...item, show: !item.show });
-            }}
-          >
-            {columnDropdownText}
-          </DropDown>
-        )}
+        {allowTogglingColumns &&
+          (enableDropDownControls ? (
+            <MultiSelectDropDown
+              buttonAriaLabelClosed={`Open column selection menu`}
+              buttonAriaLabelOpen={`Close column selection menu`}
+              itemSelectionLegend={`Select columns to display`}
+              selectAllAriaLabel={`Select all columns`}
+              resetToDefaultAriaLabel={`Reset to default columns`}
```

Comment:
Is it typical to have custom Aria Labels as inputs to components? I don't have a better suggestion off the top of my head but this seems cumbersome. Not asking for changes just want to put some thought into the best way to manage this kind of thing across the whole library of components.